parsers/typescript.c

```diff
@@ -518,35 +536,37 @@ CTAGS_INLINE void parseString(const int c, tokenInfo *const token, const char qu
 			*prev = c;
 			result->status = PARSER_NEEDS_MORE_INPUT;
 		}
-		else
-		{
-			result->status = PARSER_FAILED;
+		else result->status = PARSER_FAILED;
```

Comment:
```
else
   result->status = PARSER_FAILED;
```
I prefer putting a newline char after `else`.

Reply:
I agree that it would look nicer but it's easier to forget about curly braces when you'd like to add another statement there. One line kinda protects you from that mistake.

Reply:
In that csae,
```
					if (!parsingType)
					{
						member = newToken ();
						copyToken (member, token, false);
						member->scope = scope;
						if (visibility)
							member->accessKeyword = visibility;
						else
							member->accessKeyword = KEYWORD_public;
	
```
should be
```
					if (!parsingType)
					{
						member = newToken ();
						copyToken (member, token, false);
						member->scope = scope;
						if (visibility)
							member->accessKeyword = visibility;
						else member->accessKeyword = KEYWORD_public;
```,
shouldn't it?

Reply:
You got me :) The same applies to that if. I'll try to make my code more consistent.

Reply:
I searched "else" in typescript.c:
```
19 matches for "else" in buffer: typescript.c
    445:	else if (*parsed == 0)
    492:		else if (result->status == PARSER_FINISHED)
    505:	else if (state->isBlock)
    515:	else if (c == '\n')
    539:		else result->status = PARSER_FAILED;
    543:	else if (c == EOF)
    557:	else if (c == quote && *prev != '\\')
    585:	else if (c == '}')
    590:	else if (c == end)
    592:	else if ((state->curlyLevel < 1 && c == ';') || c == EOF)
    736:	else if (result.unusedChars > 0)
    738:	else
    902:						else
   1026:					else if (! parsingType && shouldEnd)
   1501:		else
   1610:					else
   1709:							else
   1808:						else
   1828:	else if (member)
```

In the most of all cases, you use stand-alone  'else' .

Reply:
This is what happens when you suddenly change your mind... :) I'll change all of them one way or the other.

Reply:
Howeer, this is a minor point.
Supporiting semicolons-less typescript input is much important, of course.
Feel free to merge the change.

Reply:
Thanks, I will fix formatting today and then merge.